### PR TITLE
Add --open param

### PR DESCRIPTION
**Pull request by vrunoa**
Automatically tries to open ```tunnel URL``` in the browser if the ```--open``` params is set.

### CODE REVIEW

**Comment by defunctzombie** [comment]
Issues with this PR:

- Won't work on windows or linux cause
- You are bumping the package version.. not your call to make what the new version is :)
- not following any of the existing project style for the new code.

This feature would be desired but this PR is not ready.

**Comment by vrunoa** [comment]
Hey @defunctzombie just changed the code to also work on linux & windows. Sorry for that. 

One question, what do you mean by:
* not following any of the existing project style for the new code.

How could I improve the code to follow the module style ?

**Comment by defunctzombie** [comment]
Take a look at the code style of other files (spacing, indentation, etc).

File: package.json [hunk 1]
```diff
@@ -2,16 +2,17 @@
...
+  "version": "1.5.2",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "git://github.com/shtylman/localtunnel.git"
   },
   "dependencies": {
-    "request": "2.11.4",
+    "debug": "0.7.4",
+    "openurl": "^1.1.0",
```
Comment (defunctzombie) [inline package.json]: also notice how this line is different than the others

Comment (vrunoa) [inline package.json]: I guess npm sort alphabetically the dependencies when I add the new ```openurl``` dependency. I also fix the spacing & indentation on ```bin/client``` source.

Comment (defunctzombie) [inline package.json]: That is not what I meant. Please see how else it differs from the other dependencies.

Comment (vrunoa) [inline package.json]: I remove the ```^```, sorry if this wasnt what you meant. I can see what's different.

File: bin/client [hunk 2]
```diff
@@ -43,7 +46,11 @@ lt_client(opt.port, opt, function(err, tunnel) {
     }
 
     console.log('your url is: %s', tunnel.url);
-
+    
+    if (argv.open) {
+      require('openurl').open(tunnel.url);
```
Comment (defunctzombie) [inline bin/client]: indentation

Comment (defunctzombie) [inline bin/client]: all requires should be at top of file

File: package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "author": "Roman Shtylman <shtylman@gmail.com>",
   "name": "localtunnel",
   "description": "expose localhost to the world",
-  "version": "1.5.1",
+  "version": "1.5.2",
```
Comment (defunctzombie) [inline package.json]: why is this still here?

File: bin/client (hunk 2 above)
Comment (vrunoa) [inline bin/client]: Sorry, I didn't see I was missing 2 blank spaces. I also move the require('openurl') to the top of the file.

File: package.json (hunk 3 above)
Comment (vrunoa) [inline package.json]: Fixed.

**Comment by vrunoa** [comment]
Commits squashed

**Comment by vrunoa** [comment]
:+1:

**Pull request merged**